TheMovieDB/TheMovieDB/MVP/Views/HomeScreenVC.swift

```diff
@@ -49,13 +62,17 @@ class HomeScreenVC: UIViewController {
             collectionView.isHidden = true
         }
         
-        facade.networkProvider.getMovies(completion: { [weak self] result in
+        Facade.facadeSingleton.facadeGetMovies(completion: { [weak self] result in
             
             self?.movies = result
             
             DispatchQueue.main.async { [weak self] in
-                self?.tableView.reloadData()
-                self?.collectionView.reloadData()
+                switch self?.deviceIdiom {
+                case .pad:
+                    self?.collectionView.reloadData()
+                default:
+                    self?.tableView.reloadData()
+                }
```

Comment:
Move this code to a separate method called load data or something like that

Reply:
Fixed!